shared_model/interfaces/permissions.hpp

```diff
@@ -98,9 +99,13 @@ namespace shared_model {
      public:
       PermissionSet();
       PermissionSet(std::initializer_list<Perm> list);
+      PermissionSet(const std::string &s);
```

Comment:
Please mark it explicit and maybe rename parameter to "bitstring" for more clarity.

Also I do not know whether or not constructing permission set from bitstring is the right abstraction for this class?

Reply:
The following code won't work then:
https://github.com/hyperledger/iroha/blob/baef0cf9b68b61678c0fb170bc97c0fae011227f/irohad/ametsuchi/impl/postgres_wsv_query.cpp#L99-L100